Mixpanel/MPABTestDesignerConnection.m

```diff
@@ -17,9 +17,8 @@
 #import "MPSwizzler.h"
 #import "MPResources.h"
 
-#define CONNECTIVITY_INDICATOR_SIZE 50
-
 NSString * const kSessionVariantKey = @"session_variant";
+static const int connectivityIndicatorSize = 50;
```

Comment:
One more minor thing, can we prefix this or do all caps so it's obvious it is a constant, something like

`kConnectivityIndicatorSize` or `CONNECTIVITY_INDICATOR_SIZE`

Why did you opt for a constant instead of checking the size of the image at run time?